src/app/clusters/thermostat-server/thermostat-server.cpp

```diff
@@ -557,10 +558,10 @@ bool emberAfThermostatClusterSetpointRaiseLowerCallback(app::CommandHandler * co
     int8_t DeadBand                          = 0;
     uint32_t FeatureMap                      = 0;
 
-    // TODO re-enable reading reaturemap once https://github.com/project-chip/connectedhomeip/pull/9725 is merged
 #ifndef FEATURE_MAP_OVERIDE
-    emberAfReadServerAttribute(endpoint, Thermostat::Id, chip::app::Clusters::Globals::Attributes::Ids::FeatureMap,
-                               (uint8_t *) &FeatureMap, sizeof(FeatureMap));
+    emberAfReadServerAttribute(aEndpointId, chip::app::Clusters::Thermostat::Id,
```

Comment:
Again, Accessors.

Reply:
Followup is fine for this if we just want to get this merged.

Reply:
as above